### PR TITLE
Move image to scratch and add manifest list

### PR DESCRIPTION
**Pull request by iuliancristea**
Towards https://github.com/eeveebank/platform-engineering/issues/1630

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,86 +11,71 @@ GIT_REPO          		:= github.com/$(VENDOR)/$(PROJNAME)
...
+GCR_REGISTRY 			:= eu.gcr.io
+GCR_REPO		  		:= $(GCR_REGISTRY)/mettle-bank
 GCLOUD_SERVICE_KEY		?="unknown"
 GCLOUD_SERVICE_EMAIL 	:= circle-ci@mettle-bank.iam.gserviceaccount.com # use mettle-bank (public registry) or eevee-bank (private registry)
-GOOGLE_PROJECT_ID		:=mettle-bank # use mettle-bank (public registry) or eevee-bank (private registry)
+GOOGLE_PROJECT_ID		:= mettle-bank # use mettle-bank (public registry) or eevee-bank (private registry)
 GOOGLE_COMPUTE_ZONE		:= europe-west2-a
 
-CIRCLE_BUILD_NUM  		?="unknown"
+CIRCLE_BUILD_NUM  		?= 3000
```
Comment (benhartley) [inline Makefile:21]: Better to have it as an intentionally "wrong" value?

File: go.mod [hunk 2]
```diff
@@ -4,6 +4,7 @@ go 1.14
 
 require (
 	github.com/aws/aws-sdk-go v1.30.4
+	github.com/certifi/gocertifi v0.0.0-20200922220541-2c3bb06c6054
```
Comment (benhartley) [inline go.mod:7]: Can we trust this?

**Review by mliu-mettle — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (iuliancristea) [reply Makefile]: I've updated it. Forgot to change it after some local testing

File: go.mod (hunk 2 above)
Comment (iuliancristea) [reply go.mod:7]: We can inspect the CA pool here https://github.com/certifi/gocertifi/blob/master/certifi.go

Comment (mliu-mettle) [reply go.mod:7]: Can use snyk to scan?

**Review by nwv-danvulpe — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (benhartley) [reply go.mod:7]: I don't know, sets of an alarm bell in my head :smile:

**Pull request merged**